lib/MiLight/MiLightClient.cpp

```diff
@@ -322,7 +322,7 @@ void MiLightClient::update(JsonObject request) {
 
   JsonVariant brightness = request[GroupStateFieldNames::BRIGHTNESS];
   JsonVariant level = request[GroupStateFieldNames::LEVEL];
-  const bool isBrightnessDefined = !brightness.isUndefined() || !level.isUndefined();
+  const bool isBrightnessDefined = !brightness.isNull() || !level.isNull();
```

Comment:
isUndefined() was an unofficial Method in ArduinoJson and was removed in further Versions. So it needs to be replaced with isNull().